docs/docs/contribute/getting-started.md

```diff
@@ -14,109 +14,193 @@ license: CC-BY-SA-4.0
 
 Learn how to set up the development environment to start working with our projects!
 
-## Code Editor ##
 
-A code editor allows you to work with code more easily. We will be using Visual Studio Code (VS Code) for this tutorial. You can download it [here](https://code.visualstudio.com/download), and install it with the default settings.
+## Setting up the C/C++ Compiler
 
-!!! note
-    You are free to use any code editor of your choice, although you will have to complete steps of this tutorial through alternate means.
+A compiler translates human-readable code into machine code. We will be setting up the C/C++ compiler so that our code written in the human-readable C++ language can be understood by the computers!
 
-## Setting up the GCC Compiler ##
+=== "Windows"
 
-A compiler translates human-readable code into machine code. We will be setting up the GCC compiler so that our code written in the human-readable C++ language can be understood by the computers!
+    First, [download Chocolatey](https://docs.chocolatey.org/en-us/choco/setup#installing-chocolatey) and install it using an [administrative shell](https://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-8.1/). Chocolatey is a package manager that simplifies the installation of the tools we will need.
```

Comment:
It will be better to avoid nested instructions (which our previous documentation suffered from) as it redirects the tutorial flow and makes it more complicated than necessary. If possible, we should just state the instructions for using an administrative shell here.

Reply:
I actually just copied the link that is on the chocolatey page. I could just remove that line and just keep "follow instructions to install Chocolatey" but since their page is not easy to read I wanted to help direct the user here.

I don't want to copy/paste the command line to install chocolatey from their page because it could change in the future and we shouldn't have to maintain that part.

Reply:
I was thinking we could move the Tip on opening the command prompt at line 96 to this segment instead.

So the instructions could be something like:
    
```
    1. Open up the Command Prompt.

    ??? Tip
        You can use the Windows Search Bar to find the Command Prompt application.
        ![Search for the Command Prompt in the Window’s search bar.](assets/search-command-prompt.png)
     
     2. Use the Command Prompt to install Chocolatey by following the instructions [here](https://docs.chocolatey.org/en-us/choco/setup#installing-chocolatey).
```

Reply:
The thing is chocolatey must use an administrative command prompt (i.e. open a prompt with admin privileges) but the rest of the commands should not (I think).